docs/components/tools.md

```diff
@@ -1065,3 +1065,22 @@ Flags:
   - `/-/ready` starts after all the bootstrapping completed (e.g object store bucket connection) and ready to serve traffic.
 
 > NOTE: Metric endpoint starts immediately so, make sure you set up readiness probe on designated HTTP `/-/ready` path.
+
+## Metrics
+
+### Metrics Exported By Bucket
```

Comment:
I think this header is redundant. The metric name and description should convey enough information.

Reply:
Done

Reply:
Looks like you removed too much. I meant that the `### Metrics Exported By Bucket` was redundant, but `## Metrics` should've stayed.

Reply:
Hi @douglascamata, a reminder about approval for this PR.